### ISSUE TITLE
ESM export is problematic

### ISSUE THREAD

**Issue by mkatrenik** [k1r0s/kaop#9]
Hi, in dist/kaop.esm.js export looks like 
```
var src = {
  createClass: main.createClass,
  ...
};
export default src;
```

and that makes `import { extend } from 'kaop'` invalid - it's not possible to destruct object in import like this, you can only do `import kaop from 'kaop'; kaop.extend ...`

This breaks `kaop-ts` bundled with esm aware bundler like parcel or browser supportig esm modules.

**Comment by k1r0s** [comment]
hi @mkatrenik ,thanks for the issue, going to take a look

**Comment by k1r0s** [comment]
@mkatrenik what do you think? https://github.com/k1r0s/kaop/pull/10

**Comment by k1r0s** [comment]
@mkatrenik I just released v4.1.2

**Comment by mkatrenik** [comment]
Works, thx! 👍
